subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/models/__init__.py

```diff
@@ -0,0 +1,26 @@
+from . import user, metadata
+
+from typing import List, Dict
+
+import matrix_benchmarking.models as models
```

Comment:
can you rename this into:
```suggestion
import matrix_benchmarking.models as matbench_models
```
to avoid (reading) confusions with eg this `from . import models`  in `horreum_lts_store.py`